javascript/ql/src/semmle/javascript/dataflow/Nodes.qll

```diff
@@ -1379,6 +1379,48 @@ module PartialInvokeNode {
     }
   }
 
+  /**
+   * A partial call that behaves like a throttle call, like `require("call-limit")(fs, limit)` or `_.memoize`.
+   * Seen as a partial invocation that binds no arguments.
+   */
+  private class ThrottleLikePartialCall extends PartialInvokeNode::Range, DataFlow::CallNode {
+    int callbackIndex;
+
+    ThrottleLikePartialCall() {
+      callbackIndex = 0 and
+      (
+        this = LodashUnderscore::member(["throttle", "debounce", "once", "memoize"]).getACall()
+        or
+        this =
+          DataFlow::moduleImport(["call-limit", "lodash.debounce", "lodash.throttle", "debounce"])
```

Comment:
```suggestion
          DataFlow::moduleImport(["call-limit", "debounce"])
```
These should be included in `LodashUnderscore::member` (which is the main the reason that module exists).